deploy/uninstall/uninstall.yaml

```diff
@@ -20,8 +70,6 @@ spec:
         - --force
         env:
         - name: POD_NAMESPACE
-          valueFrom:
-            fieldRef:
-              fieldPath: metadata.namespace
+          value: longhorn-system
```

Comment:
POD_NAMESPACE is normally pointed to the namespace pod is running. Change the name to e.g. `LONGHORN_NAMESPACE`.

Reply:
Modifying this leads to update manager image immediately. Is that ok?

Reply:
Sure